mitodl/open-discussions
# Added command and tasks to backpopulate a default channel's subscribers

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #1064

#### What's this PR do?
Adds a command and tasks that backpopulates a default channel

#### How should this be manually tested?
Run `./manage.py  backpopulate_default_channel CHANNEL_NAME` and verify the tasks run and complete in celery
